app/src/main/java/org/kiwix/kiwixmobile/zim_manager/library_view/LibraryFragment.kt

```diff
@@ -61,7 +63,7 @@ import org.kiwix.kiwixmobile.zim_manager.library_view.adapter.LibraryListItem
 import org.kiwix.kiwixmobile.zim_manager.library_view.adapter.LibraryListItem.BookItem
 import javax.inject.Inject
 
-class LibraryFragment : BaseFragment() {
+open class LibraryFragment : BaseFragment(), BaseFragmentActivityExtensions {
```

Comment:
shouldn't this interface only be on OnlineLibraryFragment?

Reply:
It uses the `override onBackPressed` implementation here. Why should this fragment not implement the BaseFragmentActivityExtensions?

Reply:
I didn't think it was reporting this to anyone because the LibraryFragment specifically is only used in Old navigation from ZimManageActivity